src/components/common/safe-html/SafeHtml.tsx

```
@@ -0,0 +1,20 @@
+    as?: 'h1' | 'h2' | 'h3' | 'h4' | 'p' | 'span' | 'div';
+}
+
+const CONFIG = {
+    ALLOWED_TAGS: ['b', 'strong', 'i', 'em', 'br', 'span'],
+    ALLOWED_ATTR: ['class'],
+};
+
+export const SafeHtml: React.FC<SafeHtmlProps> = memo(({ html, as: Component = 'div', className, ...props }) => {
+    const sanitizedHtml = DOMPurify.sanitize(html, CONFIG);
+
+    return <Component className={className} dangerouslySetInnerHTML={{ __html: sanitizedHtml }} {...props} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)